src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { Person } from '../../types';
+import { PersonInfo } from '../PersonInfo/PersonInfo';
+
+type Props = {
+  people: Person[];
+};
+
+export const PeopleTable: React.FC<Props> = ({ people }) => {
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Sex</th>
+          <th>Born</th>
+          <th>Died</th>
+          <th>Mother</th>
+          <th>Father</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        {people.map(person => {
+          const father = people.find(
+            personItem => personItem.name === person.fatherName,
+          );
+          const mother = people.find(
+            personItem => personItem.name === person.motherName,
+          );
```

Comment:
Using `find` inside the `map` function for each person can lead to performance issues, especially with large datasets, as it results in a quadratic time complexity. Consider creating a map or dictionary of people by name before the `map` function to quickly access the mother and father by name.